Makefile

```diff
@@ -24,6 +27,15 @@ release: build build-init
 	docker system prune -f
 	docker build $(DOCKER_BUILD_FLAGS) -t amazon/aws-for-fluent-bit:init-latest -f ./scripts/dockerfiles/Dockerfile.init-release .
 
+.PHONY: local
+local: build
```

Comment:
As Wesley commented earlier in discussion you may be able to use these environment variables to determine if you are in Code Build.

Maybe something like: `CODEBUILD_BUILD_ARN`. If this exists you know you're in the pipeline (presumably).

https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html

Reply:
this means it can be a single statement: RUN if -z blah then set env

Reply:
Maybe use somehting like: if [-z $CODEBUILD_BUILD_ARN]; then export DOCKER_BUILD_FLAGS="LOCAL=true; $DOCKER_BUILD_FLAGS"; else export DOCKER_BUILD_FLAGS="LOCAL=false; $DOCKER_BUILD_FLAGS" fi

Reply:
Ah yea... I think you can't read env vars inside of DOckerfiles? I just tried testing that out myself and it didn't work. 

Otherwise I meant that inside the Dockerfile you could do:
```
RUN if [ -z "$CODEBUILD_BUILD_ARN" ];then go env -w GOPROXY=direct;fi

```

Reply:
Mine doesn't seem to work, so use @matthewfala suggestion.